python-main.js

```diff
@@ -27,8 +27,12 @@ function pythonEditor(id) {
     ACE.getSession().setTabSize(4); // Tab=4 spaces.
     ACE.getSession().setUseSoftTabs(true); // Tabs are really spaces.
     ACE.setFontSize(editor.initialFontSize);
+    $("#request-repl")[0].style.fontSize = "" + editor.initialFontSize + "px";
```

Comment:
Doesn't hterm offer a way to edit the font? I'd prefer to use the hterm API if available.

Reply:
Yes hterm does which I am using, but that is for the blue bar that appears above hterm which is just css which has to be modified.

Reply:
Right, sorry. Apart from moving it outside of `pythonEditor` this line is fine then.